### PR TITLE
[PHPMD] The document for Sider's recommended set of rules.

### PR DESCRIPTION
**Pull request by sleeeto**
We provide the document for Sider's recommended set of rules.

### CODE REVIEW

File: docs/getting-started/recommended-rules.md [hunk 1]
```diff
@@ -27,5 +27,6 @@ Sider applies the recommended ruleset to the analysis tools below. Please follow
 - [ESLint](../tools/javascript/eslint.md#default-configuration-for-eslint)
 - [Flake8](../tools/python/flake8.md#default-configuration-for-flake8)
 - [PMD](../tools/java/pmd.md#default-configuration-for-pmd)
+- [PHPMD](../tools/php/phpmd.md#default-configuration-for-phpmd)
 - [PHP_CodeSniffer](../tools/php/code-sniffer.md#default-configuration)
```
Comment (suetsuguk) [inline docs/getting-started/recommended-rules.md:31]:
```suggestion
- [PHP_CodeSniffer](../tools/php/code-sniffer.md#default-configuration)
- [PHPMD](../tools/php/phpmd.md#default-configuration-for-phpmd)
- [PMD](../tools/java/pmd.md#default-configuration-for-pmd)
```

Reorder list items alphabetically.

Comment (sleeeto) [reply docs/getting-started/recommended-rules.md]: OK, I fixed it.

File: docs/tools/php/phpmd.md [hunk 2]
```diff
@@ -17,6 +17,12 @@ hide_title: true
 
 To start using PHPMD, enable it in your [repository settings](../../getting-started/repository-settings.md).
 
+## Default Configuration for PHPMD
+
+Sider provides our [recommended ruleset](https://github.com/sider/runners/blob/HEAD/images/phpmd/sider_recommended_phpmd.xml) for PHPMD.
+This configuration is used when you have no PHPMD configurations or `sider.yml` in your repository.
```
Comment (suetsuguk) [inline docs/tools/php/phpmd.md:23]: According to the source code, our recommended file is used when the `rule` option is not specified. How about making the same sentence as cppcheck.

```suggestion
This configuration is used when the `rule` option is not specified in `sider.yml`.
```

Comment (suetsuguk) [reply docs/tools/php/phpmd.md:23]: And also:
- In the options table, the default value of the `rule` should be `sider_recommended_phpmd.xml`
- The link "If omitted, Sider uses the default configuration." in the `rule` section should be `sider_recommended_phpmd.xml`

Comment (sleeeto) [reply docs/tools/php/phpmd.md]: Thanks, I fixed it.

**Review by suetsuguk — APPROVED** [review]
LGTM. Thank you.

**Pull request closed without merging**